src/status_im/chat/events.cljs

```diff
@@ -165,6 +157,24 @@
                           message))
                       messages)))))
 
+(defn- init-console-chat
+  [{:keys [chats] :accounts/keys [current-account-id] :as db} existing-account?]
+  (if (chats const/console-chat-id)
+    {:db db}
+    (cond-> {:db (-> db
+                     (assoc :new-chat sign-up/console-chat)
```

Comment:
Rely on `assoc` optional args to make this a single assoc call?

Reply:
We still need the `update` as well, so it won't be single call without the `->` threading macro anyway, but at least those two `assoc` can be compacted to one, good point.